daemon/src/engine/qcommon/q_shared.h

```diff
@@ -1523,32 +1502,17 @@ void         ByteToDir( int b, vec3_t dir );
 	  MEMSTREAM_FLAGS_ERR = BIT( 1 ),
 	};
 
-// helper struct for reading binary file formats
-	typedef struct memStream_s
-	{
-		byte *buffer;
-		int  bufSize;
-		byte *curPos;
-		int  flags;
-	}
-
-	memStream_t;
-
-	memStream_t *AllocMemStream( byte *buffer, int bufSize );
-	void        FreeMemStream( memStream_t *s );
-	int         MemStreamRead( memStream_t *s, void *buffer, int len );
-	int         MemStreamGetC( memStream_t *s );
-	int         MemStreamGetLong( memStream_t *s );
-	int         MemStreamGetShort( memStream_t *s );
-	float       MemStreamGetFloat( memStream_t *s );
 
 //=============================================
 
 #include <algorithm>
+
+// Could be replaced with std::max
 #ifndef MAX
 #define MAX(x,y) std::max((x), (y))
 #endif
 
+// Could be replaced with std::min
```

Comment:
Sure, I'm worried a trivial search and replace will add unneeded, or forget, parenthesis.